fairseq/trainer.py

```diff
@@ -576,6 +585,14 @@ def get_meter(self, name):
             return None
         return self.meters[name]
 
+    def meters_to_device(self, device):
```

Comment:
How many meters there are?
This is sending one tensor at a time, which can be slow.
If there are not many meters, and this gets called only at app init, it might be OK.

Reply:
there are about 10 or so. Not that big of a deal imho, this function is only called initially once.

Reply:
OK, there will be a new API in https://github.com/pytorch/xla/pull/1377 which does batching.